Library/Homebrew/service.rb

```diff
@@ -267,5 +277,28 @@ def to_systemd_unit
 
       unit + options.join("\n")
     end
+
+    # Returns a `String` systemd unit timer.
+    # @return [String]
+    sig { returns(String) }
+    def to_systemd_timer
+      timer = <<~EOS
+        [Unit]
+        Description=Homebrew generated timer for #{@formula.name}
+
+        [Install]
+        WantedBy=timers.target
+
+        [Timer]
+        Persistent=true
+        Unit=#{@formula.service_name}
```

Comment:
```suggestion
```

https://www.freedesktop.org/software/systemd/man/systemd.timer.html#Unit=

Documentation says this parameter does not need to be set as it defaults to the timer name.

Reply:
Yeah I figured better safe than sorry